src/main/java/com/direwolf20/buildinggadgets/common/packets/Packets.java

```diff
@@ -33,6 +32,8 @@ public static void register() {
         INSTANCE.registerMessage(id++, SetRangePacket.class, SetRangePacket::encode, SetRangePacket::decode, SetRangePacket.Handler::handle);
         INSTANCE.registerMessage(id++, SetModePacket.class, SetModePacket::encode, SetModePacket::decode, SetModePacket.Handler::handle);
         INSTANCE.registerMessage(id++, UndoPacket.class, UndoPacket::encode, UndoPacket::decode, UndoPacket.Handler::handle);
+        INSTANCE.registerMessage(id++, RequestTemplatePacket.class, RequestTemplatePacket::encode, RequestTemplatePacket::new, RequestTemplatePacket::handle);
```

Comment:
Please use the decode method like all the other packet classes have.

Reply:
Why do they have it like that? This seems weird to me and I'd favor changing them, if must be.

Reply:
It's currently the common standard used across most forge mods as the registerMessage calls it the decode method. It's to make it clear.

Reply:
Even so, I'd request removing the extra Handler classes. They are just a legacy from older forge versions where you needed them.